tests/fixtures/mockbin/mockbin

```diff
@@ -77,29 +77,36 @@ if prog == 'tito':
 elif prog in ['koji', 'brew']:
     parser = argparse.ArgumentParser()
     subparsers = parser.add_subparsers()
+
     parser_listpkgs = subparsers.add_parser('list-pkgs')
     parser_listpkgs.add_argument('--tag')
     parser_listpkgs.add_argument('--package')
     parser_listpkgs.add_argument('--quiet', action='store_true')
-    parser_listtagged = subparsers.add_parser('list-tagged')
```

Comment:
This is still used in `koji_old.yml`, but we never call that in our tests now. It feels like we'll end up with bit-rot if we drop the tests?

Reply:
I added it back and switch the downstream test repo to use the old syntax. So that there is now a mix of the two throughout the tests until we drop diff_package_tags